src/loadsave_gui.cpp

```diff
@@ -15,6 +18,34 @@
 #include "rev.h"
 #include "sprite_store.h"
 
+/** Sorting order for a list of savegames. */
+using SortBy = std::function<bool(const PreloadData&, const PreloadData&)>;
```

Comment:
Generally, std::function should be avoided due to its high overheads and general slowness https://wolchok.org/posts/cxx-trap-2-std-function/